autocti/charge_injection/plot/imaging_ci_plotters.py

```diff
@@ -310,6 +316,132 @@ def figures_1d(
                 ),
             )
 
+    def figures_1d_data_binned(

```

Comment:
This looks exactly the same as the method from the other class. Can we move this functionality to Plotter?